src/components/data/lang/en.json

```diff
@@ -1,19 +1,19 @@
 {
     "key": "en",
-    "title": "Blubber",
-    "subtitle": "A docker-compose.yml builder for wikibase-docker",
+    "title": "wikibase-docker Config Wizard",
```

Comment:
I'm almost against calling it this.
It makes it sound like this is the only way you can use the docker images.
At the end of the day this only generates a docker-composer file...